tests/openvino/utils_tests.py

```diff
@@ -51,6 +51,7 @@
     "llama": "fxmarty/tiny-llama-fast-tokenizer",
     "m2m_100": "hf-internal-testing/tiny-random-m2m_100",
     "opt": "hf-internal-testing/tiny-random-OPTModel",
+    "opt125m": "facebook/opt-125m",
```

Comment:
The test should be relatively fast because we don't do the inference (only once at tracing). We just convert w/ no-copy and quantize weights.

Reply:
It is hard to find the smaller model because the method expects weight tensor should contain at least 128 columns to have 1 group for group-wise quantization.

Reply:
OK let's keep it then !